apiserver/facades/controller/charmrevisionupdater/state.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
```

Comment:
To be consistent with juju precendent, this file should be called interface.go.  state brings to mind juju/state.

Reply:
Renamed to interface.go. (Though I see there are quite a number of `state.go`'s around the place too. It is kinda intended to bring to mind juju/state, as it's an interface of a subset of `*state.State`).